src/goto-programs/interpreter.cpp

```diff
@@ -31,8 +41,41 @@ Function: interpretert::operator()
 
 void interpretert::operator()()
 {
+  show=true;
+  message->status() << "0- Initialize:" << messaget::eom;
+  initialise(true);
```

Comment:
At some point it was agreed to use American English across the code base, so please use "initialize".